### ISSUE TITLE
[Feature request] Add option to load Newick file from URL

### ISSUE THREAD

**Issue by danielhjacobs** [AABoyles/TidyTree#46]
On https://cdcgov.github.io/TidyTree/app/, add a text input and submit button that on click fetches (Fetch API) and builds the tree from that url text input. Add it as a query parameter.

Also, related, add code that if you were to go to https://cdcgov.github.io/TidyTree/app/?file=https://example.com/input.nwk would build the tree for https://example.com/input.nwk

**Comment by AABoyles** [comment]
This is a great idea! Off the top of my head, it might not work extremely well because of CORS issues, but it'll probably work out-of-the-box for files that are stored in GitHub at least. I'll take a crack at it.

**Comment by danielhjacobs** [comment]
Yeah, I'll admit my use-case is to use TidyTree with a few toggle-able options on a bioinformatics website I'm working on, and then for those who want more freedom to change more options to add an "Open with Tidy Tree" option that opens https://cdcgov.github.io/TidyTree/app/?file=https://example.com/path/to/user-newick-file.nwk in a iframe. Since I control https://example.com/, I can give it an Access-Control-Allow-Origin of https://cdcgov.github.io/ when a requests comes in from https://cdcgov.github.io/ using my server configuration.

I just think that will be better than copying all the toggle-able options and the interface itself from that website.

**Comment by danielhjacobs** [comment]
I opened #47 for this.
